homeassistant/components/husqvarna_automower/lawn_mower.py

```diff
@@ -50,6 +52,24 @@ async def async_setup_entry(
         AutomowerLawnMowerEntity(mower_id, coordinator) for mower_id in coordinator.data
     )
 
+    platform = entity_platform.async_get_current_platform()
+    if platform is not None:
```

Comment:
This will always be true.

Reply:
Done in https://github.com/home-assistant/core/pull/117269/commits/77ca240ba62a65fcecfa819ad6a95d7f66f21036